src/mscorlib/src/System/Diagnostics/Eventing/EventSource.cs

```diff
@@ -250,6 +254,10 @@ namespace System.Diagnostics.Tracing
     /// </remarks>
     public partial class EventSource : IDisposable
     {
+#if FEATURE_CORECLR && FEATURE_PAL   
+        private static EventLogger persistent_Listener = new EventLogger();
```

Comment:
Now we check for an environment variable- Complus_EnableEvent to be defined before logging events